lib/ui/recommendation-row.js

```diff
@@ -194,8 +223,11 @@ RecommendationRow.prototype = {
     recommendation's URL.
     */
     let elem = createElement(row.doc, 'description', 'url');
-    elem.textContent = row._unescape(row._get('result.url'));
-    elem.setAttribute('data-url', row._get('result.url'));
+    let url = row._unescape(row._get(row.resultType == TYPES.WIKIPEDIA ?
+                                     'enhancements.wikipedia.url' :
+                                     'result.url'));
```

Comment:
I think it's best to stick with the more verbose style from lines 212-216, just for consistency